plugins/modules/mysql_user.py

```diff
@@ -118,12 +119,13 @@
     version_added: '0.1.0'
 
 notes:
-   - "MySQL server installs with default login_user of 'root' and no password. To secure this user
-     as part of an idempotent playbook, you must create at least two tasks: the first must change the root user's password,
-     without providing any login_user/login_password details. The second must drop a ~/.my.cnf file containing
-     the new root credentials. Subsequent runs of the playbook will then succeed by reading the new credentials from
-     the file."
+   - "MySQL server installs with default login_user of 'root' and no password.
```

Comment:
How about using `I(...)` for `login_user` and `login_password`? And `C(...)` for `'root'`?

Reply:
Sounds good! Fixed, thanks!